test/phone.js

```diff
@@ -62,5 +64,9 @@ describe('phone', () => {
     // TODO(thosakwe): Add a test for auto-finding the zip code of a repeat caller
     // Note from thosakwe: I'll be able to add such a test once we have some kind of persistence
     // set up.
+
+    // TODO: Add test for invalid zip code
```

Comment:
Thanks for adding these!  Do you mind adding your name `TODO(bfaloona): xx` to them?  Doesn't mean you must do them, I just find it useful to have names attached to todos :)